subworkflows/nf-core/vcf_annotate_ensemblvep/meta.yml

```diff
@@ -1,54 +1,52 @@
+# yaml-language-server: $schema=https://github.com/nf-core/modules/tree/master/subworkflows/yaml-schema.json
```

Comment:
@mashehu this is new, what it the idea behind it?

Reply:
this allows your editor to apply the schema to your meta.yml and give you hints about errors (and also what is expected for certain elements)

Reply:
I have this error in my VSCode:
```
Unable to parse content from 'https://github.com/nf-core/modules/tree/master/subworkflows/yaml-schema.json': Parse error at offset 7.YAML(768)
```

Reply:
that line is actually not in the current template, we actually point to the raw file. If you could please update this and any other occurrences of this in any meta.yml (also for modules) to: `https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json`

Reply:
I hope I'll get a sticker for it

Reply:
nope, this is no contribution to nf-core/infrastructure 
![image](https://user-images.githubusercontent.com/6169021/234803217-507ed261-7253-4503-9222-9df34237db0f.png)

Reply:
How come this PR is not infrastructure https://github.com/nf-core/modules/pull/3306
It's a @mirpedrol / @mashehu PR
This is pure infrastructure.

Reply:
https://github.com/nf-core/modules/pull/3340